### DISCUSSION TITLE
Is the document up to date?

### DISCUSSION THREAD

**Discussion by rongxin-yu** [ibis-project/ibis#6008] [General]
Hi,

I started to try out the IBIS project and really like it. I ran a lot of SQL queries and it was hard to maintain. IBIS seems to come to rescue. 

One quick note I would like to leave is that the document seems to be outdated. For example, it seems to me that with TimeValue t, one can do things like t.hour(), t.day(). However this is not documented on 
https://ibis-project.org/api/expressions/timestamps/#ibis.expr.types.temporal.TimeValue

I had to infer it by checking the backend matrix and then guess it.

**Comment by cpcloud** [comment]
Thanks for the opening the discussion!

I'm going to convert this to an issue as there's an actionable task here: ensure that `hour`, `minute` appear on the appropriate docs page.

**Comment by cpcloud** [comment]
Created https://github.com/ibis-project/ibis/issues/6352! Let's continue the discussion there.
